lib/utils/sessionStarter.js

```diff
@@ -36,16 +36,16 @@ const connectToMasterIpc = (port) => {
  * Start automated or interactive session based on config
  * @param {Object} suitest instance
  * @param {string} deviceId
- * @param {Object} config
+ * @param {Object} generalConfig
  */
-const bootstrapSession = async(suitest, deviceId, config) => {
+const bootstrapSession = async(suitest, {deviceId, config}, generalConfig) => {
```

Comment:
Why we have now two cofigs `generalConfig` and `config`? What the difference?

Reply:
generalConfig is more like runtime configurations. Config is configId which is comes from presets.

Reply:
I would rename `generalConfig` to something more specific, for example `suitestCtx` `suitestRuntimeCtx` e.t.c.